### PR TITLE
IBP-4029 convert germplasm tab to angular table

### PR DESCRIPTION
**Pull request by mderamos**

### CODE REVIEW

File: src/main/java/org/generationcp/middleware/domain/dms/Enumeration.java [hunk 1]
```diff
@@ -65,6 +65,14 @@ public String getDescription() {
 		return this.description;
 	}
 
+	public int getRank(){
```
Comment (darla-leafnode) [inline src/main/java/org/generationcp/middleware/domain/dms/Enumeration.java:68]: Is this change needed still?

Comment (mderamos) [reply src/main/java/org/generationcp/middleware/domain/dms/Enumeration.java:68]: Hi @darla-leafnode, it's no longer needed. I'll just revert the commit.

**Review by darla-leafnode — APPROVED** [review]
